test/acceptance/teams-controller.spec.js

```diff
@@ -5,22 +5,30 @@ const app = require('../../src/app');
 const request = supertest(app);
 
 describe('[Acceptance] - TeamsController', () => {
-  describe('GET /', () => {
-    let res;
+  describe('POST /', () => {
+    describe('When passing a team that already exists', () => {
+      let res;
+
+      beforeEach(async () => {
+        res = await request.post('/teams', {
+          name: 'flamengo',
+        });
+      });
 
-    beforeEach(async () => {
-      res = await request.get('/teams');
+      it('Should return statusCode 400', () => {
+        expect(res.statusCode).toBe(400);
+      });
```

Comment:
Alem disso aqui vc tambem pode usar um outro `it` para testar o body de resposta. O que acha?